code/src/index.css

```diff
@@ -1,13 +1,168 @@
+:root {
+  --purple: #edbfb9;
+  --orange: #ff9534;
+  --beige: #faeedd;
+  --blue: #0040a0;
+  --transparent-light: rgba(0, 0, 0, 0.2);
+  --transparent-dark: rgba(0, 0, 0, 0.5);
+}
+
+* {
+  box-sizing: border-box;
+}
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: raleway;
```

Comment:
Maybe add this one on the other font-family-row? Not sure if we should delete the others or keep them. I think it is good to have a fallback font. But not sure how that works. Do you know?